documentation/src/main/markdown/integrations/azureplugin/azurepluginreleasenotes.md

```diff
@@ -1,10 +1,22 @@
 # Release Notes for Azure DevOps Plugin
 
+## Version 10.0.0
+**Notice**
+
+The [company_name] Software Integrity Group is now [var_company_name]    
```

Comment:
@sig-cpotts I think it is good to add this here - As of version 10.0.0 our plugin will be available from a [new marketplace location](https://marketplace.visualstudio.com/items?itemName=blackduck.blackduck-detect)

_Like you updated detect-ado repo's overview.md_